modules/installation-guide/examples/che-proc_pulling-images-used-by-che-to-a-private-registry.adoc

```diff
@@ -26,21 +26,23 @@ The images _essential_ for starting a workspace are infrastructure images that a
   `quay.io/eclipse/che-plugin-metadata-broker`
 | Images for adding plug-ins to workspaces. See xref:administration-guide:che-workspaces-architecture.adoc#che-plug-in-broker_{context}[Che plug-ins broker].
 
-| `quay.io/eclipse/che-plugin-registry` +
-  `quay.io/eclipse/che-devfile-registry`
-| Plug-in and devfile registries that store metadata for workspaces. See xref:building-offline-versions-of-the-plug-in-and-devfile-registries_{context}[] for information about building appropriate images.
+| `quay.io/eclipse/che-plugin-registry` 
+| Plug-in registry that store metadata for workspaces. See xref:building-an-offline-plug-in-registry-image_{context}[].
```

Comment:
"that stores information about plugins and the runtime containers they require."

Reply:
```suggestion
| Plug-in registry that store information about plugins and the runtime containers they require. See xref:building-an-offline-plug-in-registry-image_{context}[].
```

Reply:
Subject to verb doesn't match here:
> registry that store
should be
... registry that store*s* ...